gen_cert.py

```diff
@@ -465,230 +464,6 @@ def _use_unicode_font(self, string):
-        addMapping('OpenSans-Light', 0, 0, 'OpenSans-Light')
-        addMapping('OpenSans-Regular', 1, 0, 'OpenSans-Bold')
-        addMapping('SourceSansPro-Light', 0, 0, 'SourceSansPro-Light')
-        addMapping('SourceSansPro-Light', 1, 1, 'SourceSansPro-SemiboldItalic')
-        addMapping('SourceSansPro-Regular', 0, 0, 'SourceSansPro-Regular')
-
-        styleArial = ParagraphStyle(
-            name="arial",
-            leading=10,
-            fontName='Arial Unicode',
-        )
-        styleOpenSansLight = ParagraphStyle(
-            name="opensans-light",
-            leading=10,
-            fontName='OpenSans-Light',
-        )
-        styleSourceSansPro = ParagraphStyle(
-            name="sourcesans-regular",
-            leading=10,
-            fontName='SourceSansPro-Regular',
-        )
-        styleSourceSansProLight = ParagraphStyle(
-            name="sourcesans-light",
-            leading=10,
-            fontName='SourceSansPro-Light',
-        )
-
-        # Text is overlayed top to bottom
-        #   * Issued date (top right corner)
-        #   * "This is to certify that"
-        #   * Student's name
-        #   * "successfully completed"
-        #   * Course name
-        #   * "a course of study.."
-        #   * honor code url at the bottom
-        WIDTH, HEIGHT = landscape(A4)
-        standardgray = colors.Color(0.302, 0.306, 0.318)
-
-        LEFT_INDENT = 55  # mm from the left side
-        DATE_INDENT = 45  # mm from the right side for Date
-
-        # Issued ..
-        style = styleSourceSansProLight
-        style.fontSize = 12
-        style.textColor = standardgray
-        style.alignment = TA_LEFT
-
-        paragraph_string = get_cert_date(generate_date, self.issued_date, self.locale, self.timezone)
-
-        # Right justified so we compute the width
-        width = stringWidth(paragraph_string, 'SourceSansPro-SemiboldItalic', style.fontSize) / mm
-        paragraph = Paragraph("<i><b>{0}</b></i>".format(paragraph_string), style)
-        paragraph.wrapOn(c, WIDTH * mm, HEIGHT * mm)
-        paragraph.drawOn(c, (WIDTH - DATE_INDENT - width) * mm, 159 * mm)
-
-        # Certify That
-        styleSourceSansPro.fontSize = 14
-        styleSourceSansPro.textColor = standardgray
-        styleSourceSansPro.alignment = TA_LEFT
-
-        paragraph_string = "This is to certify that,"
-
-        paragraph = Paragraph(paragraph_string, styleSourceSansPro)
-
-        paragraph.wrapOn(c, WIDTH * mm, HEIGHT * mm)
-        paragraph.drawOn(c, LEFT_INDENT * mm, 135 * mm)
-
-        #  Student name
-        # default is to use the DejaVu font for the name, will fall back
-        # to Arial if there are unusual characters
-        style = styleOpenSansLight
-        style.fontSize = 34
-        width = stringWidth(student_name.decode('utf-8'), 'OpenSans-Bold', style.fontSize) / mm
-        paragraph_string = "<b>{0}</b>".format(student_name)
-
-        if self._use_unicode_font(student_name):
```

Comment:
You should double-check, but I think this is the final remaining reference to this method.
We should be able to remove it at this point.